packages/nimble-components/src/anchor-button/template.ts

```diff
@@ -0,0 +1,54 @@
+import { html, ref, slotted, ViewTemplate } from '@microsoft/fast-element';
+import {
+    AnchorOptions,
+    endSlotTemplate,
+    FoundationElementTemplate,
+    startSlotTemplate
+} from '@microsoft/fast-foundation';
+import type { AnchorButton } from '.';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<AnchorButton>,
+AnchorOptions
+> = (context, definition) => html<AnchorButton>`
+    <a
+        class="control"
+        part="control"
+        ?disabled="${x => x.disabled || !x.href}"
```

Comment:
This is necessary for proper disabled styling. We use the shared button styles, which expect the element with class `control` to have the `disabled` attribute when it should be styled as such. In the case of the button, the `control` element is a `button`, but for the anchor button it is an `a`. So we're setting this not for functionality, but for style.

Reply:
We talked offline and we're going to make it so that we trigger styling off of the host element's `disabled` attribute instead the `.control`'s. Then I won't have to do this.